src/components/icon/icons/hexagon.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function Hexagon({
+  text,
+  fontFamily,
+  stroke,
+  fill,
+  fontFill,
+  strokeWidth,
+  height,
+  width,
+  viewBox,
+  style: styleProp,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+  return (
+    <svg width={width} height={height} viewBox={viewBox} style={style} {...rest}>
+      <path d="M 55,0 L 105,30 L 105,85 L 55,110 L 5,85 L 5,30 z" fill={fill} strokeWidth="0" />
```

Comment:
Does not seem to be possible to set the border in a different color (requirement from design)

Probably you want to use `fill` for the *body* and `stroke` for the *border*

Maybe this can be done if you enable `strokeWidth` and set `stroke={stroke}`?